src/components/Navigation/TopBar/TopBar.styles.tsx

```diff
@@ -1,6 +1,8 @@
 import { Button } from '@equinor/eds-core-react';
 import { tokens } from '@equinor/eds-tokens';
 
+import { UnreadRedDot } from 'src/components/Navigation/TopBar/Notifications/Notifications';
```

Comment:
This import could be shortened. 
 './Notifications/Notifications' i think (not 100% sure so did not do suggestion 🙈😂 )